src/search/landmarks/landmark_count_heuristic.cc

```diff
@@ -222,6 +224,7 @@ void LandmarkCountHeuristic::generate_preferred_operators(
     assert(successor_generator);
     vector<OperatorID> applicable_operators;
     successor_generator->generate_applicable_ops(state, applicable_operators);
+    vector<OperatorID> preferred_operators_conjunctive;
```

Comment:
When we remove the hierarchy, we can remove all three vector and just directly call `set_preferred(operators[op_id])`